### PR TITLE
Support feedback

### PR DESCRIPTION
**Pull request by ghost**
fixing browserify to output multiple scripts
changing post method to AJAX to avoid an ugly success page redirect

### CODE REVIEW

File: app/scripts/satisfaction.js [hunk 1]
```diff
@@ -1,11 +1,20 @@
-window.onload = function () {
+jQuery(document).ready(function($) {
```
Comment (meddy) [inline app/scripts/satisfaction.js]: Why are you referencing jQuery globally, instead of importing it since you are using a module system?

File: app/scripts/satisfaction.js [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
   var satisfaction = getQueryVariable('satisfaction');
   var origin = getQueryVariable('origin');
   document.getElementById('result').value = satisfaction;
   document.getElementById('origin').value = origin;
   document.getElementById('ip').value = myip;
-  document.getElementById('satisfaction').submit();
-}
+});
+$( window ).load(function() {
```
Comment (meddy) [inline app/scripts/satisfaction.js]: Why are you referencing jQuery with a $ here now?
Why are you wrapping window in jQuery?

File: app/views/feedback.jade [hunk 3]
```diff
@@ -8,7 +8,8 @@ html
 
     link(rel='stylesheet' href='styles/main.css')
     script(type="text/javascript" src="https://l2.io/ip.js?var=myip")
-    script(type="text/javascript" src="/scripts/satisfaction.js")
+    script(type="text/javascript" src="https://code.jquery.com/jquery-2.2.3.min.js")
+    script(type="text/javascript" src="/scripts/satisfaction.min.js")
```
Comment (meddy) [inline app/views/feedback.jade]: I'm probably missing some context here, but why did you need a separate bundle?
Why are you loading this script in the head instead of at the end of the body like other app scripts?

File: buildConfig/tasks/browserify.js [hunk 4]
```diff
@@ -5,25 +5,33 @@ import buffer from 'vinyl-buffer';
...
 
-    return b.bundle()
-      .pipe(source('scripts.min.js'))
-      .pipe(buffer())
-      .pipe($.sourcemaps.init({loadMaps: true}))
-        .pipe($.uglify())
-        .on('error', gutil.log)
-      .pipe($.sourcemaps.write('./'))
-      .pipe(gulp.dest(paths.distPaths.scripts));
+    return es.merge.apply(null, tasks);
```
Comment (meddy) [inline buildConfig/tasks/browserify.js]: Why do this?
You could just execute the browserify gulp command with different source file arguments.

Comment (ghost) [inline buildConfig/tasks/browserify.js]: as far as I can tell, that will mush all your source files into one.  I want two.

File: app/views/feedback.jade (hunk 3 above)
Comment (ghost) [inline app/views/feedback.jade]: there are things in the main script that break my page and vice versa, I have no good reason for sticking my scripts in the head other than it's a habit

File: app/scripts/satisfaction.js (hunk 1 above)
Comment (ghost) [inline app/scripts/satisfaction.js]: I need all the things wrapped in this (document).ready function to happen before the things in the window.load function.  I made it less ugly for you now.

File: app/views/feedback.jade (hunk 3 above)
Comment (meddy) [inline app/views/feedback.jade]: Okay. IMO you are complicating your build process to support a one off script. This script doesn't need any of the transformations that browserify is doing. It doesn't import or export any modules, use ES6 synax, or even warrant minification. Why not put this script in a directory outside of the one the browserify task references, and just include it in this file? Then your browserify task doesn't have to do twice as much as work.

As far the location of scripts (head vs. body), you should just try and be consistent. Technically putting it in the body will help eliminate the browser locking briefly as it parses your scripts before rendering the body.

File: app/scripts/satisfaction.js [hunk 5]
```diff
@@ -1,11 +1,22 @@
-window.onload = function () {
+$(document).ready(function() {
```
Comment (meddy) [inline app/scripts/satisfaction.js]: $.ready and window.load are events that can fire at different times when the page is loading. Are you sure you won't run into a race condition? It might make sense to put everything in $.ready. Also you can use `$(function () {...});` as a short hand to ready.

Comment (ghost) [inline app/scripts/satisfaction.js]: that works, is simpler, and looks better.  You're a cool guy Max.

File: app/support/satisfaction.js [hunk 6]
```diff
@@ -1,11 +1,19 @@
-window.onload = function () {
+$(function() {
   var satisfaction = getQueryVariable('satisfaction');
   var origin = getQueryVariable('origin');
   document.getElementById('result').value = satisfaction;
   document.getElementById('origin').value = origin;
   document.getElementById('ip').value = myip;
-  document.getElementById('satisfaction').submit();
-}
+  var formData = $('form').serialize();
```
Comment (mowchan) [inline app/support/satisfaction.js]: Could be argued that this `formData` variable is not necessary considering you're only using it once. You could just plug `$('form').serialize()` into the `ajax` call.

File: app/styles/main.scss [hunk 7]
```diff
@@ -189,4 +188,12 @@ h3,h4, h5, h6 {
...
 #invisible-button {
   display: none;
-}
\ No newline at end of file
+}
+
+.thanks {
+  background: #454545;
+  color: #fff;
+  background-size: cover;
```
Comment (mowchan) [inline app/styles/main.scss]: Is there a reason to have this?

Comment (ghost) [inline app/styles/main.scss]: not anymore, but there was at one point...

File: buildConfig/tasks/browserify.js [hunk 8]
```diff
@@ -26,4 +26,4 @@ module.exports = function(gulp, $, reload, gutil) {
       .pipe($.sourcemaps.write('./'))
       .pipe(gulp.dest(paths.distPaths.scripts));
   }
-}
+}
```
Comment (meddy) [inline buildConfig/tasks/browserify.js]: non blocking: new line

**Comment by meddy** [comment]
🍰

**Comment by mowchan** [comment]
:cake:

File: app/support/satisfaction.js [hunk 9]
```diff
@@ -1,11 +1,18 @@
-window.onload = function () {
+$(function() {
   var satisfaction = getQueryVariable('satisfaction');
   var origin = getQueryVariable('origin');
   document.getElementById('result').value = satisfaction;
   document.getElementById('origin').value = origin;
   document.getElementById('ip').value = myip;
```
Comment (TomWoodward) [inline app/support/satisfaction.js:6]: where does this `myip` come from?

**Comment by TomWoodward** [comment]
:cake:

File: app/support/satisfaction.js (hunk 9 above)
Comment (ghost) [inline app/support/satisfaction.js:6]: from here.  
 script(type="text/javascript" src="https://l2.io/ip.js?var=myip")
